### DISCUSSION TITLE
Boomerang Uk

### DISCUSSION THREAD

**Discussion by cohendjdjd** [iptv-org/community#374] [Show and tell]
### Channel ID

BoomerangUK.uk

### Stream URL

https://list.iptvcat.com/my_list/s/18f1251906c9357a80d50896ba7c658d.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://list.iptvcat.com/my_list/s/18f1251906c9357a80d50896ba7c658d.m3u8 (offline)
<img width="534" alt="image" src="https://user-images.githubusercontent.com/7253922/232254255-252ded7e-4e71-4d17-8f00-6653415f9b99.png">
